docs/snmp/snmp_framework.md

```diff
@@ -16,7 +16,7 @@ The framework defines:
 Built-in MIB mappings are defined in the configuration file available on the SR Linux's file system:
 
 ```bash
-cat /opt/srlinux/snmp/snmp_files_config.yaml
+# cat /opt/srlinux/snmp/snmp_files_config.yaml
```

Comment:
will remove the `#` sign. The commands are meant to be copied, and the prompt chars will prevent this